xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1557 @@
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.ICommunicationObject.EndOpen : IAsyncResult -&gt; unit&#xA;override this.System.ServiceModel.ICommunicationObject.EndOpen : IAsyncResult -&gt; unit" Usage="serviceChannelProxy.System.ServiceModel.ICommunicationObject.EndOpen result" />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.ICommunicationObject.EndOpen(IAsyncResult ^ result) = System::ServiceModel::ICommunicationObject::EndOpen;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.ICommunicationObject.EndOpen(System.IAsyncResult)</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="result" Type="System.IAsyncResult" />
+      </Parameters>
+      <Docs>
+        <param name="result">The <see cref="T:System.IAsyncResult" /> that is returned by a call to the <see cref="Overload:System.ServiceModel.ICommunicationObject.BeginOpen" /> method.</param>
+        <summary>Completes an asynchronous operation to open a communication object.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Faulted">
+      <MemberSignature Language="C#" Value="event EventHandler System.ServiceModel.ICommunicationObject.Faulted;" />
+      <MemberSignature Language="ILAsm" Value=".event class System.EventHandler System.ServiceModel.ICommunicationObject.Faulted" />
+      <MemberSignature Language="DocId" Value="E:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#ICommunicationObject#Faulted" />
+      <MemberSignature Language="VB.NET" Value="Custom Event Faulted As EventHandler Implements System.ServiceModel.ICommunicationObject" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.ICommunicationObject.Faulted : EventHandler " Usage="member this.System.ServiceModel.ICommunicationObject.Faulted : System.EventHandler " />
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.ICommunicationObject.Faulted</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the communication object first enters the faulted state.</summary>
+        <remarks>To be added.</remarks>

```

Comment:
Should this have the remarks about it being explicitly implemented and needing to be cast?

Reply:
Unsure why the tool was unable to add the "explicit implementation" text. But I can investigate and submit it later.